lib/drives/pages/drives_page.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_app/drives/pages/create_drive_page.dart';
+
+class DrivesPage extends StatefulWidget {
+  const DrivesPage({super.key});
+
+  @override
+  State<DrivesPage> createState() => _DrivesPageState();
+}
+
+class _DrivesPageState extends State<DrivesPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: const Center(child: Text('Drives')),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          Navigator.of(context).push(
+            MaterialPageRoute(builder: (context) => const CreateDrivePage()),
+          );
+        },
+        backgroundColor: Colors.blue,
```

Comment:
this should use the primary color (`Theme.of(context).colorScheme.primary`)

Reply:
i changed that